Clutch/BundleDumpOperation.m

```diff
@@ -269,6 +269,12 @@ - (void)main {
         {
             NSString *_localPath = [originalBinary.binaryPath stringByReplacingOccurrencesOfString:_application.bundleContainerURL.path withString:@""];
             
+            //Move binary to correct path on iOS 9.2+
+            if ([_application.bundleContainerURL.path hasPrefix:@"/private/var/containers/Bundle/Application/"]) {
+                NSString* iOS92BundleContainerURL = [_application.bundleContainerURL.path stringByReplacingOccurrencesOfString:@"/private" withString:@""];
```

Comment:
`NSString *` <- asterisk before the variable not after the type.

Someday this area of code should be using `NSRegularExpression` because `-[NSString stringByReplacingOccurrencesOfString]` replaces all occurrences and we only want to replace the beginning or first match.